data/lua/core/interface.lua

```diff
@@ -36,6 +37,50 @@ if wesnoth.kernel_type() == "Game Lua Kernel" then
 		moving_unit:to_map(to_x, to_y)
 		wesnoth.wml_actions.redraw{}
 	end
+	---Fakes the move of a unit satisfying the given filter to position x, y.
+    ---Usable only during WML actions.
+	---Todo : maybe implement with C++.
+	---@param to_x integer
+    ---@param to_y integer
+    ---@param u unit
+	---@param queued boolean
+    function wesnoth.interface.move_unit_fake_queue(u, to_x, to_y, queued)
```

Comment:
We don't need any documentation comments here. Move the documentation comments to a stub definition of `add_movement` in utils/emmylua/wesnoth/units.lua.